source/copilot.txt

```diff
@@ -0,0 +1,80 @@
+.. _vsce-copilot:
+
+==================================
+MongoDB Github Copilot Participant
+==================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+
+|vsce-full| includes a MongoDB Github Copilot Participant to assist in
+using `Github Copilot <https://github.com/features/copilot>`__ with your
+MongoDB deployments. Through Github Copilot's chat feature, users with
+|vsce| can interact with their MongoDB clusters and generate code with
```

Comment:
Let's also include that they can ask questions about their database collection schema, and ask MongoDB documentation questions that the participant will provide an answer to with specific docs URLs as a reference.

Reply:
Done!